From 906df53424e1b95ff72af509a05bbf048072a933 Mon Sep 17 00:00:00 2001
From: Dongdong Liu <liudongdong3@huawei.com>
Date: Fri, 30 Jun 2023 15:37:36 +0800
Subject: doc: fix syntax in hns3 guide

[ upstream commit 059268551587f6fd929951dd7f5706bbdc46abcd ]

'::' doesn't provide pre-formatted text without an empty line after it,
so fixes it.

Fixes: cdf6a5fbc540 ("doc: add runtime option examples to hns3 guide")
Cc: stable@dpdk.org

Signed-off-by: Dongdong Liu <liudongdong3@huawei.com>
---
 doc/guides/nics/hns3.rst | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/guides/nics/hns3.rst b/doc/guides/nics/hns3.rst
index 82fa79dd73..172fcdb29f 100644
--- a/doc/guides/nics/hns3.rst
+++ b/doc/guides/nics/hns3.rst
@@ -92,6 +92,7 @@ Runtime Configuration
   ``common``.
 
   For example::
+
   -a 0000:7d:00.0,rx_func_hint=simple
 
 - ``tx_func_hint`` (default ``none``)
@@ -112,6 +113,7 @@ Runtime Configuration
   ``common``.
 
   For example::
+
   -a 0000:7d:00.0,tx_func_hint=common
 
 - ``dev_caps_mask`` (default ``0``)
@@ -124,6 +126,7 @@ Runtime Configuration
   Its main purpose is to debug and avoid problems.
 
   For example::
+
   -a 0000:7d:00.0,dev_caps_mask=0xF
 
 - ``mbx_time_limit_ms`` (default ``500``)
-- 
2.23.0

